codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/customizations/PythonServerCodegenDecorator.kt

```diff
@@ -194,6 +195,31 @@ class PyTypedMarkerDecorator : ServerCodegenDecorator {
     }
 }
 
+/**
+ * Copies the stubgen scripts to the generated crate root.
+ *
+ * The shell script `stubgen.sh` runs a quick build and uses `stubgen.py` to generate mypy compatibile
+ * types stubs for the project.
+ */
+class AddStubgenScriptDecorator : ServerCodegenDecorator {
+    override val name: String = "AddStubgenScriptDecorator"
+    override val order: Byte = 0
+
+    override fun extras(codegenContext: ServerCodegenContext, rustCrate: RustCrate) {
+        val runtime_crates_path = codegenContext.runtimeConfig.runtimeCratesPath()
+        val stubgen_python_location = "$runtime_crates_path/aws-smithy-http-server-python/stubgen.py"
+        val stubgen_python_content = File(stubgen_python_location).readText(Charsets.UTF_8)
+        rustCrate.withFile("stubgen.py") {
+            writeWithNoFormatting("$stubgen_python_content")
+        }
+        val stubgen_shell_location = "$runtime_crates_path/aws-smithy-http-server-python/stubgen.sh"
+        val stubgen_shell_content = File(stubgen_shell_location).readText(Charsets.UTF_8)
+        rustCrate.withFile("stubgen.sh") {
+            writeWithNoFormatting("$stubgen_shell_content")
+        }
```

Comment:
Variables should be camelCase

Reply:
Lol.. I am writing Python Rust and Kotlin at the same time 😬 

I'll fix this.